File: Modul 5/Soal 2/PRAK402-2210817310006-Ahmad Muhsin Zein.py
angka = int(input())
for a in range(1, angka + 1) :
    if (a % 2 == 1) :
        if a != angka :
            print(a, end=' ')
        else :
            print(a)
    elif (a % 2 == 0) and a == angka :
        print()
b = angka 
while b > 0 :
    if (b % 2 == 0) :
        print(b, end=' ') ; b -= 2
    else : b -= 1

File: Modul 5/Soal 2/PRAK402-2210817310006-Ahmad Muhsin Zein.c
#include <stdio.h>

int main()
{
    int angka,a,b;
    scanf("%d", &angka);
    for(a = 1;a < angka + 1;a++){
        if (a % 2 == 1){
                if (a != angka){
                printf(" %d ", a);}
        }
        else if(a % 2 == 0 && a == angka){
                printf("\n");
    }
    }
    b = angka;
    while(b>0){
        if(b%2 == 0){
                printf(" %d ",b); b -= 2;
        }
        else{
            b -= 1;
        }
    }
}
